src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ExtenderProvidedPropertyAttribute.cs

```diff
@@ -4,47 +4,46 @@
 
 namespace System.ComponentModel
 {
-    /// <internalonly/>
     /// <summary>
-    ///    <para>
-    ///       ExtenderProvidedPropertyAttribute is an attribute that marks that a property
-    ///       was actually offered up by and extender provider.
-    ///    </para>
+    /// ExtenderProvidedPropertyAttribute is an attribute that marks that a property
+    /// was actually offered up by and extender provider.
     /// </summary>
     [AttributeUsage(AttributeTargets.All)]
     public sealed class ExtenderProvidedPropertyAttribute : Attribute
     {
         /// <summary>
-        ///     Creates a new ExtenderProvidedPropertyAttribute.
+        /// Creates a new ExtenderProvidedPropertyAttribute.
         /// </summary>
         internal static ExtenderProvidedPropertyAttribute Create(PropertyDescriptor extenderProperty, Type receiverType, IExtenderProvider provider)
         {
-            ExtenderProvidedPropertyAttribute e = new ExtenderProvidedPropertyAttribute();
-            e.ExtenderProperty = extenderProperty;
-            e.ReceiverType = receiverType;
-            e.Provider = provider;
+            ExtenderProvidedPropertyAttribute e = new ExtenderProvidedPropertyAttribute
+            {
+                ExtenderProperty = extenderProperty,
+                ReceiverType = receiverType,
+                Provider = provider
+            };
             return e;
```

Comment:
As long as you're changing this, why define the local at all, rather than just `return new ExtenderProvidedPropertyAttribute ...`?

Reply:
Got it